brillig_vm/src/registers.rs

```diff
@@ -1,42 +1,23 @@
 use crate::Value;
 use serde::{Deserialize, Serialize};
 
-/// Registers will store field element values during the
-/// duration of the execution of the bytecode.
 #[derive(Debug, Clone, PartialEq, Eq)]
 pub struct Registers {
-    pub inner: Vec<Value>,
-}
-
-impl IntoIterator for Registers {
-    type Item = Value;
-    type IntoIter = RegistersIntoIterator;
```

Comment:
Unused. I don't think anyone needs to iterate register space?

Reply:
Right, I think if it's not used internally then it's not needed

Reply:
A register iterator was used previously when setting the brillig outputs in the BrilligSolver, but that has changed now and we can remove this